cves/kernel/CVE-2014-2851.yml

```diff
@@ -339,9 +349,10 @@ stacktrace:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  any_stacktraces:
-  stacktrace_with_fix:
-  note:
+  any_stacktraces: true
+  stacktrace_with_fix: true
+  note: |
```

Comment:
Could you explain what the stacktrace was and how it points out the issues that the vulnerability produces?

Reply:
I'll provide more info.